src/Networking/NexusMods.Networking.NexusWebApi.NMA/ApiKeyMessageFactory.cs

```diff
@@ -75,7 +75,7 @@ public ValueTask SetApiKey(string apiKey)
         {
             Name = result.Data.Name,
             IsPremium = result.Data.IsPremium,
-            IsSupporter = result.Data.IsSupporter,
```

Comment:
So I'm guessing that IsSupporter data isn't available in the response?

Reply:
The "supporter" status is deprecated.